### PR TITLE
Sort recipe list by enabled & last_updated

### PR DESCRIPTION
**Pull request by brittanystoroz**
I have a feeling we might decide against sorting by enabled AND last updated, but this is what we originally discussed back in the day. (Since recipes are going to be edited more often while they're disabled, it might wind up making sense to nix the enabled sorting and just go with last_updated)

Fixes bug 1266480 r?

### CODE REVIEW

**Comment by mythmon** [comment]
There is silly test failure here, and a lint failure (there is a line in the migration that is too long). Besides that, it looks fine to me.

r+ after CI passes.

File: normandy/recipes/tests/test_serializers.py [hunk 1]
```diff
@@ -15,6 +15,7 @@ def test_it_works(self, rf):
         assert serializer.data == {
             'name': recipe.name,
             'id': recipe.id,
+            'last_updated': Whatever(),
```
Comment (brittanystoroz) [inline normandy/recipes/tests/test_serializers.py]: @mythmon is there a better way to fix this test? If I try to set it to something like `timezone.now()` I get a failure where `last_updated` doesn't match up: 

```
{'last_updated': '2016-06-09T19:32:08.115291Z'} != {'last_updated': datetime.datetime(2016, 6, 9, 19, 32, 8, 120002, tzinfo=<UTC>)}
```

File: normandy/recipes/migrations/0029_recipe_last_updated.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9 on 2016-06-09 19:23
```
Comment (rehandalal) [inline normandy/recipes/migrations/0029_recipe_last_updated.py]: fyi: we've been adding: 
```python
# flake8: noqa
```
to get the linter to ignore migrations

File: normandy/recipes/tests/test_serializers.py (hunk 1 above)
Comment (mythmon) [inline normandy/recipes/tests/test_serializers.py]: Can you do something like `recipe.last_updated`? I guess that would still have the type error. Maybe convert it to a string with `recipe.last_updated.isoformat()`?

File: normandy/recipes/tests/test_serializers.py [hunk 3]
```diff
@@ -15,6 +15,7 @@ def test_it_works(self, rf):
         assert serializer.data == {
             'name': recipe.name,
             'id': recipe.id,
+            'last_updated': recipe.last_updated.replace(tzinfo=None).isoformat() + 'Z',
```
Comment (brittanystoroz) [inline normandy/recipes/tests/test_serializers.py]: Took a little extra magic and is not pretty but this seems to be the only way to format the date to match O_o

Comment (mythmon) [inline normandy/recipes/tests/test_serializers.py]: Oh, that's pretty gross. I can't find an easy way to make it better though :\

Comment (rehandalal) [inline normandy/recipes/tests/test_serializers.py]: FWIW, i think `Whatever()` is a better option.

Comment (brittanystoroz) [inline normandy/recipes/tests/test_serializers.py]: I like `Whatever()` better as well. Less ugly and maybe it's not something that's super important to test the exact value of, just that it exists.

Comment (mythmon) [inline normandy/recipes/tests/test_serializers.py]: I wish there was a way to assert it was a date in the right format, without getting all ugly. I'm fine with `Whatever()` though.

**Comment by brittanystoroz** [comment]
Reverted to `Whatever()`

**Pull request merged**